logger/log_test.go

```diff
@@ -25,8 +25,6 @@ import (
 	"testing"
 )
 
-var tempRoot = filepath.Join(os.TempDir(), "nats-server")
```

Comment:
this part got introduced to make it easier cleanup the tests at some point, but `testing.TempDir()` (available since Go 1.15) already creates a folder with an easy to recognize name using the test like `TestFileLogger.../001` plus it auto cleans up, so this is a good change 👍